app/components/Checkbox/index.js

```diff
@@ -46,26 +46,23 @@ const Checkbox = ({
 
     {/* This is the control box with a check icon as children */}
     <ControlBox
-      borderWidth="1.5px" // TODO: should come from theme
-      size="14px" // TODO: should come from theme
-      borderColor={
-        isDisabled ? 'checkbox.disabled.borderColor' : 'checkbox.borderColor' // TODO: Based on theme.borderColor
-      }
+      size={0.875}
+      borderWidth={1.5}
+      borderColor="borderColor.checkbox.default"
       rounded="md"
-      _checked={
-        isDisabled
-          ? {}
-          : { bg: 'brand.500', color: 'white', borderColor: 'brand.500' } // TODO: Theme.disabled should have the styles
-      }
+      _checked={{
+        bg: 'bg.checkbox.checked',
+        color: 'white',
+        borderColor: 'borderColor.checkbox.checked',
+      }}
+      _disabled={{ borderColor: 'borderColor.checkbox.disabled' }}
+      _checkedAndDisabled={{}}
```

Comment:
this is for ?

Reply:
![image](https://user-images.githubusercontent.com/39694575/77908151-91897b80-72a8-11ea-808d-3cbf6b1ae5cf.png)

Above is the checked and disabled state, I thought we can handle this state later.